packages/osslsigncode/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=osslsigncode
+pkgver=242.c0d9569
+pkgrel=1
+pkgdesc='A small tool that implements part of the functionality of the Microsoft tool signtool.exe.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-microsoft')
```

Comment:
`blackarch-microsoft` is not a valid group. I think that we should put it in `blackarch-windows` and `blackarch-automation`

Reply:
Duh hah. fixing.